clang/test/CIR/IR/tbaa-parse.cir

```diff
@@ -0,0 +1,26 @@
+// RUN: cir-opt %s -o %t
+// RUN: cir-opt %t
+
+!s16i = !cir.int<s, 16>
+!s32i = !cir.int<s, 32>
+#fn_attr = #cir<extra({nothrow = #cir.nothrow, uwtable = #cir.uwtable<async>})>
+!rec_S = !cir.record<struct "S" {!s16i} #cir.record.decl.ast>
+#tbaa_scalar = #cir.tbaa_scalar<id = "short", type = !s16i>
+#tbaa_struct = #cir.tbaa_struct<id = "S", members = {<#tbaa_scalar, 0>}>
+#tbaa_tag = #cir.tbaa_tag<base = #tbaa_struct, access = #tbaa_scalar, offset = 0>
```

Comment:
Add CHECK lines that the these 3 `#tbaa_.*`, this should allow you to eliminate the second run of `cir-opt`.